src/view/document.js

```diff
@@ -127,6 +128,8 @@ export default class Document {
 
 		injectQuirksHandling( this );
 
+		this.on( 'keydown', ( evt, data ) => _jumpOverUiElement( evt, data, this.domConverter ) );
```

Comment:
Let's move this helper to `uielement.js` and expose it as `injectQuirksHandling()` just like for the filler.